definitions-tests/definitions_test.go

```diff
@@ -0,0 +1,74 @@
+package definitions_tests
+
+import (
+	"cuelang.org/go/cue"
+	"cuelang.org/go/cue/cuecontext"
+	"github.com/stretchr/testify/assert"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+const (
+	velaTemplatesDir   = "../vela-templates/definitions/"
+	testDataDir        = "./test-data/"
+	expectedOutputsKey = "outputs"
+	cueExtension       = ".cue"
+)
+
+func TestDefinitions(t *testing.T) {
+	// 1. traverse all definition .cue file
+	var targets []string
+	filepath.Walk(velaTemplatesDir, func(path string, info fs.FileInfo, err error) error {
```

Comment:
We can omit `/workflowstep` directory here since the workflow cases are more complicated and we can support that in the future.

Reply:
omitted `/workflowstep`  directory.